examples/poisson_3d_target_torus.py

```diff
@@ -154,14 +154,16 @@ def run_model(ncells, degree, comm=None, is_logical=False):
 
     print("Start discretization", flush=True)
     # Create computational domain from topological domain
-    Omega_h = discretize(Omega, ncells=ncells, comm=comm)
-    Omega_log_h = discretize(Omega_logical, ncells=ncells, comm=comm)
+    periodic = [False, False, False]
+    periodic_log = [False, True, True]
```

Comment:
Using different periodicity condition in the two cases does not seem correct to me...

Reply:
I agree. This however has already been the case before and I can't immediately tell why it has been like this.

Reply:
Eventually we will fix this file and make sure that the simulation works. But that it beyond the scope of this PR. Thanks for looking into this!

Reply:
Hi, I'm a bit late to the party but I wanted to say that the reason why the periodicity is like this is that in the first list `periodic = [False, False, False]` corresponds to the periodicity in the cartesian `x,y,z` coordinates which are not periodic in anyway while the `periodic_log = [False, True, True]` coresponds to the periodicity in the parameter (or logical) space which for a torus is periodic in theta and phi.